src/fprime/fpp/common.py

```diff
@@ -142,7 +142,8 @@ def execute(
             ]
         else:
             input_args.extend(["-i", ",".join(map(str, imports))] if imports else [])
-            input_args.extend(sources)
+            input_args.extend([str(source) for source in sources])
+            input_args.append(str(locations))
```

Comment:
Does locations need to come first?  I always put it first, but perhaps it does not matter.

Reply:
Addressed